codalab/apps/authenz/views.py

```diff
@@ -106,10 +112,12 @@ def post(self, request, *args, **kwargs):
         token = d['token'] if 'token' in d else None
         scopes = d['scopes'] if 'scopes' in d else None
         response_data = ValidationApi._translate_token(token, scopes)
-        return HttpResponse(json.dumps(response_data), content_type="application/json")
+        return HttpResponse(
+            json.dumps(response_data), content_type="application/json")
```

Comment:
this and the 3 pervious changes are a bit much. `"user").get(` does not help. lol 
better layout for above as well. 
```
'user': {
    'id': access_token.user.id, 
    'name': access_token.user.username
}
```